modules/combined/tests/fieldsplit_contact/2blocks3d.i

```diff
@@ -100,6 +105,10 @@
     formulation = kinematic
     system = constraint
     normal_smoothing_distance = 0.1
+    order = FIRST
```

Comment:
You shouldn't need to specify order or disp_* here (and you'll get a deprecation warning about the displacements).